app/src/main/java/com/zulip/android/networking/AsyncDevGetEmails.java

```diff
@@ -0,0 +1,72 @@
+package com.zulip.android.networking;
+
+import android.content.Context;
+import android.content.Intent;
+import android.util.Log;
+import android.widget.Toast;
+
+import com.zulip.android.ZulipApp;
+import com.zulip.android.activities.DevAuthActivity;
+import com.zulip.android.activities.LoginActivity;
+import com.zulip.android.util.ZLog;
+
+import org.apache.http.HttpStatus;
+import org.apache.http.client.HttpResponseException;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+/**
+ * Created by kunall17 on 5/31/16.
```

Comment:
Can you get rid of this comment? Zulip projects rely on commit history to document authorship.